test/test.js

```diff
@@ -1,15 +1,85 @@
 const test = require('ava').test
 const fs = require('fs')
 const path = require('path')
+const vfile = require('vfile')
+const unified = require('unified')
+const parse = require('remark-parse')
+const getSection = require('../src/utilities/get-section')
+const findSection = require('../src/utilities/find-section')
 
-// TODO Make a more useful test. This literally just tests .replace on fs.
```

Comment:
Any reason you removed this test?

Reply:
As the comment said, it was a “useless” placeholder test, so now there’s real tests I thought it could be dropped! 🤷‍♂️

Reply:
hmm. Fair.

Reply:
It asserts that `String#replace` works haha! Anyway, I can add it back?

Reply:
Nah, kill it. What is dead can never die, or something witty.